examples/rspack/src/react-env.d.ts

```diff
@@ -0,0 +1,218 @@
+// CSS modules
+type CSSModuleClasses = { readonly [key: string]: string };
+
+declare module "*.module.css" {
+	const classes: CSSModuleClasses;
```

Comment:
spaces instead of tabs?

Reply:
Addressed in https://github.com/Collinbrown95/js-lingui/commit/639d7a7998bb06f0fe2251e4839a06c2a4023263